src/utils/chartJs/CustomTooltip.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import Fade from '@material-ui/core/Grow';
+
+const styles = {
+  tooltip: {
+    background: 'rgba(0, 0, 0, .8)',
+    color: 'white',
+    borderRadius: '4px',
+    pointerEvents: 'none',
+  },
+  tooltipKey: {
+    display: 'inline-block',
+    width: '10px',
+    height: '10px',
+    marginRight: '10px',
+  },
+  table: {
+    borderCollapse: 'collapse',
+  },
+};
+
+function CustomTooltip({ classes, tooltipModel }) {
+  const titleLines = tooltipModel.title || [];
+  const bodyLines = tooltipModel.body
+    ? tooltipModel.body.map(bodyItem => bodyItem.lines)
+    : [];
+  const footerLines = tooltipModel.footer || [];
+
+  return (
+    <Fade in={tooltipModel.opacity === 1}>
+      <div
+        className={classes.tooltip}
+        style={{
+          padding: `${tooltipModel.yPadding}px ${tooltipModel.xPadding}px`,
+        }}
+      >
+        <table className={classes.table}>
+          <thead>
+            {titleLines.map(title => (
+              <tr>
+                <th>{title}</th>
+              </tr>
+            ))}
+          </thead>
+          <tbody>
+            {bodyLines.map((body, i) => {
+              const { backgroundColor, borderColor } = tooltipModel.labelColors[
+                i
+              ];
+
+              return (
+                <tr>
+                  <td>
+                    <span
+                      style={{
+                        background: backgroundColor,
+                        borderColor,
+                      }}
+                      className={classes.tooltipKey}
+                    />
+                    {body}
+                  </td>
+                </tr>
+              );
+            })}
+          </tbody>
+          <tfoot>
+            {footerLines.map(footer => (
+              <tr>
+                <td dangerouslySetInnerHTML={{ __html: footer }} />
+              </tr>
+            ))}
+          </tfoot>
+        </table>
```

Comment:
so, in summary, this table is removed, and replace with the footer function, plus a little more to show the date of the datapoint

Reply:
This is a case where thinking about the future is not a good idea. Simply build the minimum.  I imagine future custom tooltips to be a copy-and-change of your custom tooltip.  You are doing the hard work of making the first one.